app/_src/using-mesh/managing-ingress-traffic/gateway-api.md

```diff
@@ -262,19 +271,146 @@ spec:
 and then reference it in your `Gateway`:
 
 ```yaml
-apiVersion: gateway.networking.k8s.io/v1beta1
+apiVersion: gateway.networking.k8s.io/v1
 kind: Gateway
 metadata:
   name: kuma
   namespace: default
 spec:
   gatewayClassName: kuma-cross-mesh
+  listeners:
+  - name: proxy
+    port: 8080
+    protocol: HTTP
 ```
 
-### Multi-zone
+### Multi-zone Deployments
 
+{% if_version lte:2.6.x %}
 Gateway API isn't supported with multi-zone deployments, use {{site.mesh_product_name}}'s [`MeshGateways`](/docs/{{ page.version }}/using-mesh/managing-ingress-traffic/builtin-listeners)/[`MeshHTTPRoute`](/docs/{{ page.version }}/policies/meshhttproute)/
 [`MeshTCPRoute`](/docs/{{ page.version }}/policies/meshtcproute) instead.
+{% endif_version %}
+{% if_version gte:2.7.x %}
+The Gateway API supports multi-zone deployments, but with some limitations:
+
+- Gateway API resources like `Gateway`, `ReferenceGrant`, and `HTTPRoute` must be created in non-global zones. They are kubernetes resources and therefore are not known to {{ site.mesh_product_name }}.
```

Comment:
```suggestion
- Gateway API resources like `Gateway`, `ReferenceGrant`, and `HTTPRoute` must be created in non-global zones}. They also don't appear directly in the {{ site.mesh_product_name }} GUI, only the {{ site.mesh_product_name }} resources they manage do.
```

Not sure what they are "not known to Kuma" means?

Reply:
I actually removed this sentence as it didn't make sense to me anymore.